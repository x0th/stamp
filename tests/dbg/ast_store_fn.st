Object test = fn(param1, param2) { param1 }
