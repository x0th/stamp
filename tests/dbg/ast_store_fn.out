STDOUT:
----------------------
Context:
Callable = Callable-hash
Char = Char-hash
Int = Int-hash
List = List-hash
Object = Object-hash
String = String-hash
false = false
if = if
true = true
----------------------
TokSList()
  TokStore()
    TokObject(Object)
    TokValue(test)
    TokSend()
      TokSend()
        TokSend()
          TokSend()
            TokObject(Callable)
            TokMessage(clone)
              TokValue(test)
          TokMessage(store_param)
            TokValue(param1)
        TokMessage(store_param)
          TokValue(param2)
      TokMessage(store_body)
        TokSList()
          TokObject(param1)
Object-hash
----------------------
Context:
Callable = Callable-hash
Char = Char-hash
Int = Int-hash
List = List-hash
Object = Object-hash
String = String-hash
false = false
if = if
true = true
----------------------
STDERR:
